libr/lang/p/python/core.h

```diff
@@ -0,0 +1,15 @@
+/* radare - LGPL - Copyright 2017-2019 - pancake, xvilka, aronsky */
+
+#ifndef _PY_CORE_H
+#define _PY_CORE_H
+
+#include <r_core.h>
+#include "common.h"
+
+extern RCore *core;
```

Comment:
Doesn't seem to be used at all, just skimmed through code. Are you sure it is still required? Seems like some leftovers.

Reply:
It's actually used by most of the code around it (there are references to `core->lib` in all the functions that build the different plugins. However, the `extern` keyword is definitely not needed, so I removed it.